File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        #Define values
        currentCount = 0
        maxCount = 0
        #Iterate arrays
        for num in nums:
            if num == 1:
                currentCount+1
            else:
                currentCount = 0
            
            if currentCount > maxCount:
                maxCount = currentCount

        return maxCount

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        //Array to store results
        int currentCount = 0;
        int maxCount = 0;
        //iterate throug array
        for(int i = 0;i < nums.length; i++){
            if(nums[i] == 1){
                currentCount++;
            } else if(nums[i] == 0){
                currentCount = 0;
            }
            
            if(currentCount > maxCount){
                maxCount = currentCount;
            }
            
        }
        return maxCount;
    }
}